docs/content/guides/dagster/recommended-project-structure.mdx

```diff
@@ -0,0 +1,44 @@
+
+  - In this example, we keep all assets together in the [`project_fully_featured/assets/`](https://github.com/dagster-io/dagster/tree/master/examples/project_fully_featured/project_fully_featured/assets) directory. It is useful because you can use <PyObject object="load_assets_from_package_module" /> or <PyObject object="load_assets_from_modules" /> to load assets into your repository, as opposed to needing to add assets to the repository every time you define one. It also helps collaboration as your teammates can quickly navigate in the codebase to find the core business logic (i.e. assets) regardless of their familiarity to the specific repository.
+
+- Keep schedules ans sensors that target a particular job together.
+
+  - In this example, we put sensors and schedules in [`jobs.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/jobs.py). When we build sensors or schedules from a job, they are considered as policies for when to trigger a particular job. So, keeping all the policies together with jobs makes it clear when and how certain jobs are being launched.
+
+  - Note: Certain sensors like [run status sensors](/concepts/partitions-schedules-sensors/sensors#run-status-sensors) can listen to multiple jobs and do not trigger a job. We recommend keeping these sensors in the repository definition as they are often for alerting and monitoring at the repository level.
+
+- Make resources reusable and share them across jobs or asset groups.
+
+  - In this example, resources (e.g., database connections, Spark sessions, API clients, and I/O managers) are grouped in the [`resources/__init__.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/\__init\_\_.py) file, where they are bound to configuration sets which vary based on environment. In complex projects, we find it very useful to make resources reusable and configured with pre-defined values via <PyObject object="configured" />. So when developing a project, instead of trying to find their own way to model external components, teammates can all quickly get stuff done using a pre-defined resource set, or make changes to resources that are shared among the team.
+
+  - This patten also helps you to easily execute jobs in different environment without code changes. In this example, we dynamically define repository based on the deployment in [`repository.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/repository.py) and are able to keep all code the same across testing, local development, staging, and production. Read more about our recommendations in [Transitioning Data Pipelines from Development to Production](/guides/dagster/transitioning-data-pipelines-from-development-to-production)
+
+- TODO: What to do with ops/graphs/jobs???
```

Comment:
I think we decided from the recommended project structure to have an `ops.py` and a `jobs.py`. I can see graphs fitting in with either ops or jobs--we don't necessarily need to address graphs though, we could leave it up to the user. 

Definitely makes sense to me to recommend having all of the ops exist in one file though, for ease of importing and reusing across multiple jobs.

Reply:
A medium-sized organization could easily have hundreds of ops, which could mean thousands of lines of code, and I think that would be unwieldy to have in one file.